content/blog/alternative-to.md

```diff
@@ -36,8 +37,8 @@ Rogue echo gym trainer is considered a gold standard in fitness circles, and for
 
 But since Rogue Echo Gym Timer comes with specific hardware, you have to spend a fortune to have it set in your fitness training area. But does that mean you have got no options other than this timer? Well, no. [Captime](http://onelink.to/captime) is one of the best alternatives to the Rogue Echo gym Timer.
 
-[Captime](http://onelink.to/captime) is a mobile app that you can install on your iPhone or iPad. 
-<Cta/>
```

Comment:
Call to action is not in place in blog article ? where it is now ?

Reply:
It is implemented directly in the blog/[id].vue file